modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1732,10 +1746,15 @@ public Expression textTranslateAnchor() {
                 "symbol-spacing", getId());
         symbolizer.getOptions().put("repeat", String.valueOf(symbolSpacing));
 
-        // text max angle
-        // layer.getTextMaxAngle();
-        // symbolizer.getOptions().put("maxAngleDelta", "40");
-
+        // text max angle - only for line placement
+        // throw MBFormatException if point placement
+        if(labelPlacement instanceof LinePlacement){
+            // will be true if line placement, false else.
+            symbolizer.getOptions().put("followLine", String.valueOf(followLine));
```

Comment:
Is there any way this can ever be false? From what I can see, the only time labelPlacement is a LinePlacement is when we have `symbol-placement = "line"`, in which case `followLine` always is true.